### DISCUSSION TITLE
Add GIF image format

### DISCUSSION THREAD

**Discussion by tom-combet** [PrestaShop/PrestaShop#40140] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

I can upload GIF images in BO but they are not animated in FO.

### Expected behavior

They should be animated in FO.

### Specs

Add an option to generate GIF images only when a GIF is uploaded

### Steps to reproduce

1. Upload GIF image (in a product page for example).
2. Check in FO, it should be static.

### PrestaShop version(s) where the bug happened

8.0.0

### PHP version(s) where the bug happened

7.4

**Comment by sLorenzini** [comment]
Hello,

I reproduce the issue with PrestaShop version 8.0.0 / 1.7.8.6 / 1.7.7.8.
However it is not a bug but a feature to implement. The .gif files are regenerated to .png or .jpg

![gif-format](https://user-images.githubusercontent.com/2854834/180460719-e7a12876-ba8a-4a7b-b667-f31c00b0dd6e.png)

The Product Team will take it into consideration for future developments. 

Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [contacting a professional](https://www.prestashop-project.org/support/) to help you create it. 

If you or anyone else is willing to [contribute this feature](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/) to the PrestaShop Project, the teams will be happy to help by ensuring that it fits well within the project. Please let us know if that is the case!

Thank you
